content/FLASHDeconv/FLASHDeconvWorkflow.py

```
@@ -85,57 +85,35 @@ def process_uploaded_files(uploaded_files):
-                        uploaded_files = [uploaded_files]
-
-                    # opening file dialog and closing without choosing a file results in None upload
-                    process_uploaded_files(uploaded_files)
-                    st.success("Successfully added uploaded files!")
-                else:
-                    st.warning("Upload some files before adding them.")
+        _, c2, _ = st.columns(3)
+        if c2.form_submit_button("Add files to workspace", type="primary"):
+            if uploaded_files:
+                # A list of files is required, since online allows only single upload, create a list
+                if type(uploaded_files) != list:
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)